src/plugins/intel_cpu/src/graph.cpp

```diff
@@ -320,8 +336,40 @@ static std::tuple<std::vector<NodePtr>, std::vector<size_t>> ExtractExecutableNo
                            std::move(executableSyncNodesInds));
 }
 
+void Graph::Configure(const std::shared_ptr<const ov::Model>& network,
+                      const GraphContext::CPtr ctx,
+                      const VecMemoryDescs& inputDescriptors,
+                      const bool zeroCopyOutputs) {
+    OPENVINO_ASSERT(status == Status::NotReady, "Invalid graph status");
+
+    context = ctx;
+
+    Replicate(network, inputDescriptors, zeroCopyOutputs);
+
+    InitGraph();
+}
+
+void Graph::Allocate() {
```

Comment:
Configure could also be just "ReplicateAndInit"

Reply:
I think we have to do the following:
Configure -> InitGraph (the existing method with this name may be renamed to something uglier or les transparent since it's a private method thus not a part of the interface). The second stage can be renamed to "commitToExecutable" or "commitToExec". It would be even better if there was only one way (two stage initialization) to create a graph. Introducing the changes we're discussing, we allow the graph to be constructed in two ways, in one call "CreateGraph" and in two calls "InitGraph" and "CommitToExec". Which makes the graph interface more complex.